src/Core/Managed/Shared/Extensibility/Metrics/Metric.cs

```diff
@@ -1,13 +1,12 @@
-﻿namespace Microsoft.ApplicationInsights.Extensibility
+﻿namespace Microsoft.ApplicationInsights.Extensibility.Metrics
```

Comment:
Beta's are to get customer feedback and make sure we don't have major bugs in new functionality.
If we have to make a breaking change, in new functionality only, to fix something then we can.
I don't know that tweaking a namespace is that important but I am open to hear why it should be renamed vs. left as is.